doc/content/getting-started/se-hands-on-guide/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "SENSOR EDITION HANDS ON TUTORIALS"
```

Comment:
`hands-on` in the title, instead of `hands on`

Reply:
Done